wetterdienst/provider/dwd/observation/download.py

```diff
@@ -48,11 +48,10 @@ def _download_climate_observations_data(remote_file: str) -> BytesIO:
     return BytesIO(__download_climate_observations_data(remote_file=remote_file))
 
 
-@payload_cache_five_minutes.cache_on_arguments()
 def __download_climate_observations_data(remote_file: str) -> bytes:
 
     try:
-        zip_file = download_file_from_dwd(remote_file)
+        zip_file = download_file(remote_file, ttl=CacheExpiry.FIVE_MINUTES)
     except InvalidURL as e:
         raise InvalidURL(
             f"Error: the station data {remote_file} could not be reached."
```

Comment:
I'd suggest using the corresponding ZipFileSystem from FSSPEC here.

Reply:
Ah, good catch. Thank you. Either I overlooked this, or fsspec was not ready in the past yet. I can look at this once again.

Reply:
Thank you for your contribution #504. I just integrated it into this patchset.